Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml

```diff
@@ -301,7 +310,7 @@
   - type: MeleeWeapon
     damage:
      groups:
-       Brute: 5
+       Brute: 10
```

Comment:
Was was this changed?

Reply:
due to the current combat system changes making it practically impossible to kill anything, 10 may be a bit much, but specifically the runner has less damage than even a basic drone or burrower, that's far too low for the current combat system

Reply:
il accept it.